txscript/stack.go

```diff
@@ -132,6 +133,25 @@ func (s *Stack) PopInt() (*big.Int, error) {
 	if err != nil {
 		return nil, err
 	}
+
+	if s.verifyMinimalData && len(so) > 0 {
```

Comment:
This code is identical, so please refactor it into a function that takes the byte array as a parameter.